boards/arm/gd32e103v_eval/doc/index.rst

```diff
@@ -0,0 +1,212 @@
+.. _gd32e103v_eval:
+
+GigaDevice GD32E103V-EVAL
+#########################
+
+
+Overview
+********
+
+The GD32E103V-EVAL board is a hardware platform that enables design and debug
+of the GigaDevice C103V Cortex-M4F High Performance MCU.()
+
+The GD32E103VB features a single-core ARM Cortex-M4F MCU which can run up
+to 120-MHz with flash accesses zero wait states, 128kiB of Flash, 32kiB of
+SRAM and 80 GPIOs.
+
+.. image:: img/gd32e103v_eval.png
+     :width: 800px
+     :align: center
+     :alt: gd32e103v_eval
+
+
+Hardware
+********
+
+- USB interface with mini-USB connector
+- 4 user LEDs
+- 4 user push buttons
+- Reset Button
+- ADC connected to a potentiometer
+- 2 DAC channels
+- GD25Q16 2Mib SPI Flash
+- AT24C02C 2KiB EEPROM
+- 3.2 TFT LCD (320x240)
+- PCM1770 Stereo DAC with Headphone Amplifier
+- GD-Link interface
+
+  - CMSIS-DAP swd debug interface over USB HID.
+
+- 2 CAN port(support CAN-FD)
+
+  - Because of the bug in hardware, this function was removed, if you need CAN function, please check `gd32c103`.
```

Comment:
This function is not available in this board due to hardware issues, please check ``GD32C103``.